### ISSUE TITLE
OWIntegrate: Split into two outputs?

### ISSUE THREAD

**Issue by stuart-cls** [Quasars/orange-spectroscopy#310]
Thinking about #307, I wonder if it would be a good idea to split the output into Spectra + Peak Parameters[1] and get rid of the "Output as metas" checkbox altogether. I think this is more consistent with what most Widgets do. 

The only thing I'm unsure of is if this would result in extra memory usage in the "Output as metas" case.

[1] ...would need a better name...

**Comment by markotoplak** [comment]
Your suggestion would only increase memory consumption very little, because X can be reused and is therefore not copied.

**Comment by markotoplak** [comment]
And I do not think there is so much consistency: k-means outputs as meta/class, whereas PCA outputs as new table. I think these are just decisions on what we think is convenient on case by case basis.

**Comment by stuart-cls** [comment]
True, but k-means doesn't have a small checkbox that changes the output shape/contents drastically. This is the root of my suggestion. It also has a separate output of the centroids. I think since we already have the use-case that either the calculation result or the annotated data are useful, it warrants two outputs.

Anyway I'm not too excited about it, it was just a thought.
